halo2_proofs/src/plonk/prover.rs

```diff
@@ -319,6 +320,9 @@ where
                     k: params.k(),
                     current_phase,
                     advice: vec![domain.empty_lagrange_assigned(); meta.num_advice_columns],
+                    unblinded_advice: HashSet::from_iter(
+                        meta.unblinded_advice_columns.clone().into_iter(),
+                    ),
```

Comment:
We could make this shorter :)
```suggestion
                    unblinded_advice: HashSet::from_iter(meta.unblinded_advice_columns.clone()),
```